info.yml

```diff
@@ -13,35 +13,91 @@
 
 # expected fields in the configuration file for this app
 configuration:
-    name:
+
+    default_node_name:
         type: str
         description: >
-            A human readable name to display in the HDA to allow selection of this
-            set of configuration options in Houdini.
+            A default name for Alembic output nodes created in houdini.
+            Allowed characters include letters, numbers, periods, dashes, or
+            underscores.
+        default_value: sgtk_alembic
 
     work_file_template:
         type: template
         description: >
-            A reference to a template which locates a Houdini work file on disk.
-            This is used to drive the version and optionally the name of output files.
-        required_fields: [version]
-        optional_fields: [name]
+            A reference to a template which locates a Houdini work file on
+            disk.  This is used to drive the version and optionally the name of
+            output files.
+        fields: context, version, [name]
```

Comment:
The `context` field is mentioned [here](https://support.shotgunsoftware.com/entries/95442678#The%20template%20data%20type). From my understanding, it allows you to reference any fields defined by the context object available when the template is evaluated. Its also used by the nuke write node which is probably how it got here. Seems like a way to allow the template to be evaluated in different contexts without being specific about `Shot`, `Asset`, etc. here in the settings.